src/main/kotlin/net/irisshaders/lilybot/extensions/util/Tags.kt

```diff
@@ -0,0 +1,193 @@
+package net.irisshaders.lilybot.extensions.util
+
+import com.kotlindiscord.kord.extensions.DISCORD_BLURPLE
+import com.kotlindiscord.kord.extensions.DISCORD_RED
+import com.kotlindiscord.kord.extensions.checks.anyGuild
+import com.kotlindiscord.kord.extensions.checks.hasPermission
+import com.kotlindiscord.kord.extensions.commands.Arguments
+import com.kotlindiscord.kord.extensions.commands.converters.impl.string
+import com.kotlindiscord.kord.extensions.extensions.Extension
+import com.kotlindiscord.kord.extensions.extensions.ephemeralSlashCommand
+import com.kotlindiscord.kord.extensions.extensions.publicSlashCommand
+import com.kotlindiscord.kord.extensions.types.respond
+import com.kotlindiscord.kord.extensions.types.respondEphemeral
+import com.kotlindiscord.kord.extensions.utils.suggestStringMap
+import dev.kord.common.entity.Permission
+import dev.kord.core.behavior.channel.GuildMessageChannelBehavior
+import dev.kord.core.behavior.channel.createEmbed
+import dev.kord.rest.builder.message.create.embed
+import kotlinx.datetime.Clock
+import net.irisshaders.lilybot.utils.DatabaseHelper
+import net.irisshaders.lilybot.utils.getConfigPrivateResponse
+import net.irisshaders.lilybot.utils.responseEmbedInChannel
+
+class Tags : Extension() {
+	override val name = "tags"
+
+	override suspend fun setup() {
+
+		publicSlashCommand(::TagArgs) {
+			name = "tag"
+			description = "The parent command for all /tag commands"
```

Comment:
Hate to do this but I now realize that I misread what this did last time. This should probably mention the /tag-help. command and give a brief description of tags. It's not a parent command and I apologize for that.